src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -116,33 +116,30 @@ MulticopterAttitudeControl::generate_attitude_setpoint(const Quatf &q, float dt,
 	if (reset_yaw_sp) {
 		_man_yaw_sp = yaw;
 
-	} else if (math::constrain(_manual_control_setpoint.z, 0.0f, 1.0f) > 0.05f
-		   || _param_mc_airmode.get() == 2) {
+	} else if ((_manual_control_setpoint.throttle > -.9f)
+		   || (_param_mc_airmode.get() == 2)) {
```

Comment:
Side note: Would be nice to have these magic numbers (e.g. `2`), an ENUM Name!

Reply:
Yes, the problem is parameter options are not automatically an enum so we have to create a local enum and keep it in sync with the parameter metadata just to check if it's 2. Definitely a compromise.